src/options.json

```diff
@@ -0,0 +1,33 @@
+{
+  "type": "object",
+  "properties": {
+    "lessOptions": {
+      "description": "Options to pass through to `less`. (https://github.com/webpack-contrib/less-loader#examples).",
+      "anyOf": [
+        {
+          "type": "object",
+          "additionalProperties": true
+        },
+        {
+          "instanceof": "Function"
+        }
+      ]
+    },
+    "sourceMap": {
```

Comment:
We should allow only `boolean` value,  `outputSourceFiles` it is the less specificity option, we should not respect it